.circleci/config.yml

```diff
@@ -0,0 +1,64 @@
+version: 2.1
+
```

Comment:
Let's wait for Grigory fixes - this part on him.

Reply:
i've added this simple ci in aims to have our template for auto publishing feature